packages/bs-loader/index.js

```diff
@@ -83,15 +73,19 @@ function processBsbError(err) {
   return undefined
 }
 
+function getWarningsFromOutput(output) {
```

Comment:
I think this can be inlined too, it's a pretty simple function call and only used once (?)